tools/certs/README.md

```diff
@@ -1,2 +1,6 @@
 To enable https://web-platform.test:8443/, add cacert.pem to your browser as Certificate Authority.
-E.g., for Firefox, go to about:preferences and search for "certificates".
+
+For Firefox, go to about:preferences and search for "certificates".
+
+For browsers that use the Certificate Authorities of the underlying OS, such as Chrome and Safari,
+you need to adjust the OS. For macOS, go to Keychain Access and add the certificate under System.
```

Comment:
Is there any reason why it needs to be under the System keychain? Can it be under the login keychain?

Reply:
Ah, yes it can.